bundler/spec/runtime/setup_spec.rb

```diff
@@ -757,28 +757,28 @@ def clean_load_path(lp)
     context "when the user has one set" do
       before { ENV["MANPATH"] = "/foo#{File::PATH_SEPARATOR}" }
 
-      it "adds the gem's man dir to the MANPATH" do
+      it "appends the gem's man dir to the MANPATH" do
         install_gemfile <<-G
           source "#{file_uri_for(gem_repo4)}"
           gem "with_man"
         G
 
         run "puts ENV['MANPATH']"
-        expect(out).to eq("#{default_bundle_path("gems/with_man-1.0/man")}#{File::PATH_SEPARATOR}/foo")
+        expect(out).to eq("/foo#{File::PATH_SEPARATOR}#{default_bundle_path("gems/with_man-1.0/man")}")
```

Comment:
If I understand it well, this now appends (instead of original prepending) bundle man path. Is that really intended?

Reply:
You have a point. The trick as documented on the stackoverflow page is to prepend `:` to the `MANPATH`, but maybe it also works by appending it? I will try tomorrow. If appending `:` works, it will be the best behavior and it would make things fully backwards compatible.

If appending `:` doesn't work, we can still do better and keep the current behavior when the `MANPATH` is set.

I will improve this 👍.

Reply:
Confirmed prepending `:` works.

```
MANPATH=/home/postmodern/test/ruby/bundler/vendor/bundle/ruby/3.0.0/gems/kramdown-2.3.1/man man ls
No manual entry for ls
MANPATH=:/home/postmodern/test/ruby/bundler/vendor/bundle/ruby/3.0.0/gems/kramdown-2.3.1/man man ls
```

Reply:
Appending `:` seems to also work, so that should be the way to go 👍.

Reply:
I pushed a new version, it should be better now. But maybe I should add a `man ls` test to double check this works.